packages/bull-vault/src/FlashBull.sol

```diff
@@ -117,6 +117,14 @@ contract FlashBull is UniFlash {
         ethWSqueethPool = IController(IBullStrategy(_bull).powerTokenController()).wPowerPerpPool();
         ethUSDCPool =
             IController(IBullStrategy(_bull).powerTokenController()).ethQuoteCurrencyPool();
+
+        ICrabStrategyV2(IBullStrategy(_bull).crab()).approve(_bull, type(uint256).max);
+        IERC20(IController(IBullStrategy(_bull).powerTokenController()).wPowerPerp()).approve(
```

Comment:
can just use wPowerPerp since we set it?

Reply:
`wPowerPerp` is immutable, so we can't use it in the constructor when we set it also